### ISSUE TITLE
[Bug]: Popup for Authentication to Install RPM When Closing Application with Available Update

### ISSUE THREAD

**Issue by C0sm0cats** [wavetermdev/waveterm#1079]
### Current Behavior

**Description:**

When I close the application while an update is available, I receive a popup asking me to authenticate in order to install the latest version of waveterm RPM. This issue occurred with `waveterm-linux-x64-0.8.12-beta.2.pacman` and seems inappropriate for users on Manjaro, as it suggests an unnecessary installation process.

Additionally, it would be beneficial to have a toggle option to control whether we want to receive updates or not. This would give users more control over their application experience.

### Expected Behavior

- The application should close without prompting for installation or authentication.
- Users should have the option to enable or disable automatic updates through a toggle in the settings.

Thank you for your attention to this matter!

### Steps To Reproduce

1. Open the application.
2. Wait for an update to become available.
3. Close the application.
4. Observe the popup requesting authentication for RPM installation.

### Wave Version

waveterm-linux-x64-0.8.12-beta.2.pacman

### OS

manjaro

### Architecture

None

### Questionnaire

- [ ] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by esimkowitz** [comment]
There is an option to disable automatic updates, you can run `wsh setconfig autoupdate:enabled=false` and restart the app. To disable install on quit, run `wsh setconfig autoupdate:installonquit=false` and restart the app. 

As for the authentication prompt when installing an update, that's out of our control. electron-updater manages this for us and if your system asks for authentication to update the app, we prompt the user for it.

**Comment by C0sm0cats** [comment]
Thanks @esimkowitz for the clarification! I didn’t catch that command in the documentation. It really helps me understand how to manage the updates better. That answers my needs perfectly, as I was looking for a visual way to do it. Thanks again!

**Comment by C0sm0cats** [comment]
@esimkowitz  However, there’s a problem: on Manjaro, if we keep autoupdate set to true, it prompts to install the RPM version instead of the PACMAN version, and RPM isn’t compatible with Arch Linux.

**Comment by esimkowitz** [comment]
Under the hood, the pacman version is just an RPM package since we bundle the final binary, not build instructions. It shouldn't affect your install

**Comment by C0sm0cats** [comment]
Since the beta, it visually prompts to install  waveterm-linux-aarch64-0.8.12.rpm instead of waveterm-linux-aarch64-0.8.12.pacman version when on Arch, which can be confusing.

**Comment by esimkowitz** [comment]
Hmm I see, that is weird I'll take a look further. Maybe I removed something

**Comment by esimkowitz** [comment]
This might be a bug in electron-updater. The Pacman auto update support is new. Will investigate further this week

**Comment by C0sm0cats** [comment]
Yeah. it seems there is no current implementation in electron-updater for detecting the appropriate package manager for updates. This lack of detection creates confusion in selecting the correct files for auto-updates and determining which package manager to use for the update process.

**Comment by esimkowitz** [comment]
it's supposed to ignore artifacts that don't match the package type

**Comment by esimkowitz** [comment]
@C0sm0cats can you share your ~/.waveterm/waveapp.log to support@waveterm.dev? I want to see what the auto updater logs look like

**Comment by C0sm0cats** [comment]
@esimkowitz Done, you have it in your box.
